spec/README.md

```diff
@@ -214,10 +230,33 @@ This is the base of the character (e.g. the diacritic `ü` has a base of `u`, an
 ###### data.{character}.mapping.decompose
 
 Optional  
-Type: String
+Type: Object
 
 This is the character, or combination of characters used to represent the diacritic (e.g. `ö` decomposes into `oe` in German), ligature (e.g. `æ` decomposes into `ae`), or symbol (e.g. `‽` decomposes into `?!`).
 
+Each decompose value may need to be transformed into an upper, lower or title case. These entries have been included within the `decompose` values
+
+###### data.{character}.mapping.decompose.titleCase
+
+Optional  
+Type: String
+
+When a decompose value comprises two or more characters, the case of the first character may differ from the others. Include the necessary changes in this entry (e.g. `æ` and `Æ` decomposes into `Ae` in a title case).
+
+###### data.{character}.mapping.decompose.upperCase
+
+Optional  
+Type: String
+
+When a decompose value comprises two or more characters, the characters may be used in an abbreviation and require all capitals. Include the necessary changes in this entry (e.g. `æ` and `Æ` decomposes into `AE` in all upper case).
```

Comment:
A word written in upper case may not always be an abbreviation. Please specify this more exactly, e.g. `the characters may be used in a word writte in upper case`